docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,310 @@
+# This uses auto complete.
+estimator.options.default_shots = 4000
+# This does bulk update.
+estimator.options.update(default_shots=4000, resilience_level=2)
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+```python
+# Setting options after primitive initialization
+# This uses auto complete.
+options.resilience_level = 2
+estimator = Estimator(backend=backend, options=options)
+
+# Setting options after primitive initialization.
+# This does bulk update.
+estimator.set_options(shots=4000)
+```
+  </TabItem>
+</Tabs>
+
+<span id="run-method"></span>
+### Run() method
+
+The run() method accepts options in V1 only.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+In V2, the only values you can pass to `run()` are those defined in the interface.  That is, `shots` for Sampler and `precision` for Estimator. This overwrites any value set for `default_shots` or `default_precision` for the current run.
+
+```python
+# Sample two circuits at 128 shots each.
+sampler.run([circuit1, circuit2], shots=128)
+```
+
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+You can pass any options to `run()`. Because most users will only overwrite a few options at the job level, it is not necessary to specify the options category. The code below, for example, specifies `shots=1024` instead of `execution={"shots": 1024}` (which is also valid).
+```python
+estimator = Estimator(backend=backend, options=options)
+result = estimator.run(circuit, observable, shots=1024).result()
+print(f">>> Metadata: {result.metadata[0]}")
+```
+</TabItem>
+</Tabs>
+
+## Commonly used options
+
+There are many available options, but the following are the most commonly used:
+
+### Shots
+For some algorithms, setting a specific number of shots is a core part of their routines.  There are several ways to set and update shots with the primitives.
+
+<Tabs>
+  <TabItem value="primitivesV2" label="V2 Primitives">
+Shots (or precision) can be specified in multiple places in V2.  They are prioritized as follows:
+
+For any Sampler pub:
+
+1. Integer-valued shots contained in the pub
+2. The `run(...,shots=val)` value
+3. The `options.default_shots` value
+
+For any Estimator pub:
+
+1. Float-valued precision contained in the pub
+2. The `run(...,precision=val)` value
+3. The `options.default_shots` value
+4. The `options.default_precision` value
+
+Example:
+
+```python
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+# Setting shots during primitive initialization
+sampler = Sampler(backend, options={"default_shots": 4096})
+
+# Setting options after primitive initialization
+# This uses auto complete.
+sampler.options.default_shots=2000
+
+# This does bulk update.  The value for default_shots is overridden if you specify shots with run() or in the PUB.
+sampler.options.update(default_shots=1024, dynamical_decoupling={"sequence_type": "XpXm"})
+
+# Sample two circuits at 128 shots each.
+sampler.run([circuit1, circuit2], shots=128)
```

Comment:
nit - this example should include one for "Integer-valued shots contained in the pub", e.g. 

```python
# 100 shots is used for circuit1 and 200 for circuit2
sampler.run([(circuit1, None, 100), (circuit2, None, 200)])
```